rosette/rbl/rosette/tests/run-tests.ros

```diff
@@ -70,7 +104,11 @@
 (load "tests/reader-iso-8859-1.ros" 'silent)
 (load "tests/reader-utf-8.ros" 'silent)
 
-(exit (if (> failures 0) 1 0))
+;; Returning 1 or 0, not (value total-failures) on the off chance that
+;;
+;;    (= 0 (mod (value total-failures) 256))
+;;
+(exit (if (> (value total-failures) 0) 1 0))
```

Comment:
It is interesting that we don't already have a `normalize` or `nonzero` function. I learned quite a lot from this comment and one line of code.

Reply:
We might, but I couldn't guess its name. I tried `zero?` and `plus?` and `positive?` and variants.